File: _01_Languages/Kotlin/src/_01_basicos/_05_Bucles.kt
/*
 * 🧩 Enunciado:
 * Muestra los números del 1 al 10 usando:
 * - for
 * - while
 * - do-while
 */

fun main() {
    println("🔹 Bucle for:")
    for (i in 1..10) {
        print("$i ")
    }

    println("\n🔹 Bucle while:")
    var j = 1
    while (j <= 10) {
        print("$j ")
        j++
    }

    println("\n🔹 Bucle do-while:")
    var k = 1
    do {
        print("$k ")
        k++
    } while (k <= 10)
}

File: _01_Languages/CSharp/src/_01_basicos/_05_Bucles.cs
/*
 * 🧩 Enunciado:
 * Muestra los números del 1 al 10 usando:
 * - un bucle for
 * - un bucle while
 * - un bucle do-while
 */

using System;

class _05_Bucles
{
    static void Main()
    {
        Console.WriteLine("🔹 Bucle for:");
        for (int i = 1; i <= 10; i++)
            Console.Write(i + " ");

        Console.WriteLine("\n🔹 Bucle while:");
        int j = 1;
        while (j <= 10)
        {
            Console.Write(j + " ");
            j++;
        }

        Console.WriteLine("\n🔹 Bucle do-while:");
        int k = 1;
        do
        {
            Console.Write(k + " ");
            k++;
        } while (k <= 10);
    }
}
